src/groupWith.js

```diff
@@ -8,23 +8,23 @@ var _curry2 = require('./internal/_curry2');
  * @memberOf R
  * @since v0.21.0
  * @category List
- * @sig (a, a -> Boolean) -> [a] -> [[a]]
+ * @sig ((a, a) → Boolean) → [a] → [[a]]
```

Comment:
I'd actually rather keep them in the output as well, although I wouldn't mind if the font offered a ligature for the characters.